src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,56 @@
+package duke.task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * A type of duke.Task that keeps track of the date and time of a task and the task's description.
+ */
+public class Event extends Task {
+    protected LocalDate at;
+
+    /**
+     * Constructor.
+     *
+     * @param description details about the Task.
+     * @param at date/time when the Task takes place, in yyyy-mm-dd format.
+     */
+    public Event(String description, LocalDate at) {
+        super(description);
+        this.at = at;
+    }
+
+    /**
+     * Returns duke.Event's String form.
+     *
+     * @return String.
+     */
+    @Override
+    public String toString() {
+        return "[E]" + super.toString() + " (at: " + at.format(DateTimeFormatter.ofPattern("MMM d yyyy")) + ")";
```

Comment:
Perhaps would it be better if you extract the format as a static variable i.e. DATE_FORMAT = "MMM d yyyy"